### DISCUSSION TITLE
Will Vue.js be supported?

### DISCUSSION THREAD

**Discussion by sbrian26** [microsoft/fluentui#18331] [General] [answered]
Will Vue.js be supported? - Are there any efforts ongoing to port the #fluentui for direct use within vue.js projects?
[office-ui-fabric][fluentui-react] [fluentui](https://stackoverflow.com/questions/tagged/office-ui-fabric).

**Comment by gouttierre** [comment]
@sbrian26  - thanks for filling this query with us. I doubt we have upcoming plans to support Vue.js any time soon as we’re currently focused on larger coherence work items and enhancements to our performance. But let me loop in our senior pm to double check.

@paulgildea - could you confirm with the project roadmap for 2021, if we have plans to support Vue.js?

**Comment by JustSlone** [comment, marked as answer]
@sbrian26 you should be able to use the [Fluent UI Web Components](https://github.com/microsoft/fluentui/blob/master/packages/web-components/README.md) in Vue.js without too much trouble. This applies to Angular and potentially other frameworks as well. 

We won't be adding a specific Vue.js implementation but instead continue to invest in the Web Components implementation alongside React.
